.htaccess

```diff
@@ -83,4 +83,29 @@ Header set Cache-Control "max-age=31536000, public"
 # One month for css and js
 <filesMatch ".(css|js)$">
 Header set Cache-Control "max-age=2628000, public"
-</filesMatch>
\ No newline at end of file
+</filesMatch>
+
+# Enable keep-alive using .htaccess
+
+<ifModule mod_headers.c> Header set Connection keep-alive </ifModule>
+
+# For APACHE
```

Comment:
It's preferable to have this inside the httpd.conf

Reply:
I added leverage browser caching and gZIP compression above!